code/game/objects/items/theft_items.dm

```diff
@@ -171,6 +172,10 @@
 	return
 
 /obj/item/nuke_core/supermatter_sliver/can_be_pulled(user) // no drag memes
+	if(isliving(user))
+		var/mob/living/L = user
+		if(HAS_TRAIT(L, TRAIT_SUPERMATTER_IMMUNE))
+			return TRUE
 	return FALSE
```

Comment:
```suggestion
/obj/item/nuke_core/supermatter_sliver/can_be_pulled(mob/user) // no drag memes
	if(HAS_TRAIT(user, TRAIT_SUPERMATTER_IMMUNE))
		return TRUE
	return FALSE
```

Reply:
the typecasting is needed here because `user` doesn't have a `status_traits` var otherwise. at least, according to VSC

Reply:
thats why I changed the parameter. user is always a mob. its just a poorly defined proc.